packages/api/src/command/webhook/webhook.ts

```diff
@@ -133,78 +147,112 @@ export const processRequest = async (
       sendAnalytics(status);
     }
     return true;
-
-    //eslint-disable-next-line @typescript-eslint/no-explicit-any
-  } catch (error: any) {
-    // TODO: 1411 - remove when DAPI is fully discontinued
+  } catch (error) {
+    // TODO: 1411 - remove the conditional and keep the code inside it when DAPI is fully discontinued
     if (productType === Product.medical) {
       log(`Failed to process WH request: ${errorToString(error)}`);
       const status = "failure";
-      try {
-        // mark this individual WH request as failed
-        await updateWebhookRequestStatus({
-          id: webhookRequest.id,
-          status,
-        });
-      } catch (err2) {
-        log(`Failed to store failure state on WH log: ${errorToString(err2)}`);
-        capture.error(err2, {
-          extra: {
-            webhookRequestId: webhookRequest.id,
-            webhookUrl,
-            context: `webhook.processRequest.updateStatus.failed`,
-            error: err2,
-          },
-        });
-      }
```

Comment:
logic moved to `updateWhRequestWithError()`

Reply:
Diff:

<img width="1414" alt="image" src="https://github.com/metriport/metriport/assets/2132564/ec498294-ca10-4c40-af7d-c1c57337a977">